src/openrct2/Context.cpp

```diff
@@ -525,7 +525,7 @@ namespace OpenRCT2
                 LightFXInit();
             }
 
-            InputResetPlaceObjModifier();
+            InputResetModifierKeyState();
```

Comment:
As long as we rest this when the input manager is created we don't need this here.